src/libraries/System.Data.Common/src/System/Data/Common/DbConnectionStringBuilder.cs

```diff
@@ -412,6 +413,16 @@ private PropertyDescriptorCollection GetProperties()
             return propertyDescriptors;
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
+            Justification = "The type of component is statically known. This class is marked with [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]")]
```

Comment:
This isn't a valid suppression. `GetProperties` is always `RUC` because the PropertyDescriptor's Type needs to be `[DAM.All]` as well, which we can't guarantee here.

The `protected virtual void GetProperties(Hashtable)` method should be marked with `[RUC]`.